xcube/util/jsonschema.py

```diff
@@ -158,6 +168,51 @@ def to_dict(self) -> Dict[str, Any]:
         return d
 
 
+class JsonDatetimeSchema(JsonStringSchema):
+    # noinspection PyShadowingBuiltins
+    def __init__(self,
+                 min_datetime: str = None,
+                 max_datetime: str = None,
+                 format='date-time',
```

Comment:
If `format` can also be `"date"` then maybe call this class `JsonDateSchema` because the time-part is optional? Your choice.

Reply:
I would rather keep it as Datetime, regarding a date as a special case of a date-time (even if that isn't true as regards the specification syntax). More correct would be JsonDateOrDatetimeSchema, but that's horribly unwieldy.